### ISSUE TITLE
The YouTube video on the Accessibility testing page does not work because it is marked as private

### ISSUE THREAD

**Issue by BenjaminEllett** [MicrosoftDocs/windows-uwp#3993]
The YouTube video on the Accessibility testing page does not work because it is marked as private.  Here are the repro steps for this bug:

**Repro Steps**
1. Open Edge
2. Go to https://docs.microsoft.com/en-us/windows/apps/design/accessibility/accessibility-testing 

**Expected:**
The YouTube video under the **Run accessibility testing tools** heading is shown.

**What Actually Occurs**
The screen says the video is marked as private.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c7311500-9e32-4558-8108-728a9dfbff8a
* Version Independent ID: c466a80f-93b9-b58b-317d-72217a17e5b6
* Content: [Accessibility testing - Windows apps](https://docs.microsoft.com/en-us/windows/apps/design/accessibility/accessibility-testing)
* Content Source: [hub/apps/design/accessibility/accessibility-testing.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/apps/design/accessibility/accessibility-testing.md)
* Product: **reunion**
* Technology: **design**
* GitHub Login: @Karl-Bridge-Microsoft
* Microsoft Alias: **kbridge**

**Comment by Karl-Bridge-Microsoft** [comment]
@BenjaminEllett - thanks for letting us know about this. There appear to be a number of videos that have gone private recently. I've updated the link to a valid video for now. I'll update the overall topic as well.
